### PR TITLE
[minor] Support for Maximo AI Broker install

### PR DESCRIPTION
**Pull request by karol-czarnecki**
CLI support AI Broker

Proof of deployment:
image
[aibroker-cli-status.log](https://github.com/user-attachments/files/17280209/aibroker-cli-status.log)
[odh.log](https://github.com/user-attachments/files/17280191/odh.log)
[kmodels.log](https://github.com/user-attachments/files/17280194/kmodels.log)
[aibroker.log](https://github.com/user-attachments/files/17280196/aibroker.log)

### CODE REVIEW

File: tekton/src/params/install.yml.j2 [hunk 1]
```diff
@@ -651,3 +651,126 @@
   type: string
   description: controls the workload size of predict containers
   default: "small"
+
```
Comment (durera) [inline tekton/src/params/install.yml.j2:670]: To make this install file more manageable we have started breaking up the params .... e.g. see the various `tekton/src/params/install-x.yml.j2` files:

AI Broker params should be introduced in a similar way:

```
# Dependencies - IBM Maximo AI Broker
# -----------------------------------------------------------------------------
{{ lookup('template', params_src_dir ~ '/install-aibroker.yml.j2') }}
```

and then have this content in `tekton/src/params/install-aibroker.yml.j2`

File: tekton/src/params/install.yml.j2
Comment (durera) [inline tekton/src/params/install.yml.j2:1]: I don't see any corresponding update to the install pipeline itself to use these new parameters?

**Review by durera — CHANGES_REQUESTED** [review]
Also, once the changes are made can you provide the output of running the tekton build and test locally (as we've had issues recenntly with bad tekton definitions) ...

https://github.com/ibm-mas/cli/blob/master/CONTRIBUTING.md#1-building-the-tekton-definitions

File: tekton/src/params/install.yml.j2 (hunk 1 above)
Comment (karol-czarnecki) [reply tekton/src/params/install.yml.j2:670]: this is fixed

**Review by durera — CHANGES_REQUESTED** [review]
I've re-reviewed and made a few tweaks, but the tekton definitions are not valid so this is still not ready to merge ... e.g.

```
oc apply -f tekton/target/tasks/odh.yaml
Error from server (BadRequest): error when creating "tekton/target/tasks/odh.yaml": admission webhook "validation.webhook.pipeline.tekton.dev" denied the request: validation failed: non-existent variable in "$(params.mas_aibroker_tenant_name)": spec.steps[0].env[MAS_AIBROKER_TENANT_NAME]
```

**Comment by durera** [comment]
Currently re-reviewing this ...

**Review by durera — APPROVED** [review]

**Pull request merged**